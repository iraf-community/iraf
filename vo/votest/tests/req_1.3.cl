#
#  Demonstrate selection of column for tasks expecting a list.

# Set the test description string.
votest.descr = "Demonstrate selection of column for tasks expecting a list"

# Convert the data$logical to a local path.
s1 = data_url // "/sia.xml"
s2 = "file://" // data_path // "/sia.xml"
s3 = "file:///localhost" // data_path // "/sia.xml"

print ("------------------------------------------------------------------")
print ("Req  1.3:  Users shall be able to select a column in a VOTable")
print ("           for use in tasks that expect a list of values.")
print ("------------------------------------------------------------------")


fcache init
flpr 0

print (" ")
print ("    This test is demonstrated by using the @-file syntax to auto-")
print ("matically extract the image access reference  column from  an SIA")
print ("result table.")
print (" ")


# Suppress the image name to avoid a false failure because of checksum
# algorithm differences between platforms.
imstat.fields = "npix,mean,stddev,min,max"


# Execute the test commands.
print ("\nLogical Path:  data$sia.xml")			# logical path
imstat ("@data$sia.xml")

print ("\nHTTP URI:  " // s1)				# remote http URI
imstat ("@" // s1)

print ("\nFile URI:  " // s2)				# file URI
imstat ("@" // s2)

print ("\nFile URI:  " // s3)				# file URI
imstat ("@" // s3)



