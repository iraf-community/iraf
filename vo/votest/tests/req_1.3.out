------------------------------------------------------------------
Req  1.3:  Users shall be able to select a column in a VOTable
           for use in tasks that expect a list of values.
------------------------------------------------------------------
 
    This test is demonstrated by using the @-file syntax to auto-
matically extract the image access reference  column from  an SIA
result table.
 

Logical Path:  data$sia.xml
#      NPIX      MEAN    STDDEV       MIN       MAX
     262144     108.3     131.3       -1.    19936.
     262144     208.3     131.3       99.    20036.
     262144     308.3     131.3      199.    20136.
     262144     408.3     131.3      299.    20236.
     262144     508.3     131.3      399.    20336.
     262144     608.3     131.3      499.    20436.

HTTP URI:  http://iraf.noao.edu/votest//sia.xml
#      NPIX      MEAN    STDDEV       MIN       MAX
     262144     108.3     131.3       -1.    19936.
     262144     208.3     131.3       99.    20036.
     262144     308.3     131.3      199.    20136.
     262144     408.3     131.3      299.    20236.
     262144     508.3     131.3      399.    20336.
     262144     608.3     131.3      499.    20436.

File URI:  file:///iraf/iraf/vo/votest/data//sia.xml
#      NPIX      MEAN    STDDEV       MIN       MAX
     262144     108.3     131.3       -1.    19936.
     262144     208.3     131.3       99.    20036.
     262144     308.3     131.3      199.    20136.
     262144     408.3     131.3      299.    20236.
     262144     508.3     131.3      399.    20336.
     262144     608.3     131.3      499.    20436.

File URI:  file:///localhost/iraf/iraf/vo/votest/data//sia.xml
#      NPIX      MEAN    STDDEV       MIN       MAX
     262144     108.3     131.3       -1.    19936.
     262144     208.3     131.3       99.    20036.
     262144     308.3     131.3      199.    20136.
     262144     408.3     131.3      299.    20236.
     262144     508.3     131.3      399.    20336.
     262144     608.3     131.3      499.    20436.
